src/app/shared/services/utils/utils.service.spec.ts

```diff
@@ -125,4 +126,33 @@ describe('Utils service', () => {
     color = '#000000';
     expect(Utils.isColorDark(color)).toBeTruthy();
   });
+
+  it('should convert AM/PM to 24 and 24 to AM/PM', () => {
+    timeExamples.forEach(({ hour12, hour24 }) => {
+      expect(Utils.convertAmPmTo24(hour12)).toEqual(hour24);
+      expect(Utils.convert24ToAmPm(hour24)).toEqual(hour12);
+    });
+
+    expect(
+      Utils.convert24ToAmPm({
```

Comment:
If you have a dataset array, why you did not put this test cases in datasets array?